features-json/loading-lazy-attr.json

```diff
@@ -131,14 +131,14 @@
       "72":"n",
       "73":"n",
       "74":"n",
-      "75":"y",
-      "76":"y",
-      "77":"y",
-      "78":"y",
-      "79":"y",
-      "80":"y",
-      "81":"y",
-      "82":"y"
+      "75":"a #1",
+      "76":"a #1",
+      "77":"a #1",
+      "78":"a #1",
+      "79":"a #1",
+      "80":"a #1",
+      "81":"a #1"
```

Comment:
@atjn Looks like the missing comma here prevented the data in this file from being updated. Not sure if the validator missed it or if I didn't pay enough attention to it...

Reply:
I see, thanks for checking that out